### PR TITLE
Register new package Bridge.jl v0.4.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [mschauer/Bridge.jl](https://github.com/mschauer/Bridge.jl)
Release: [v0.4.0](https://github.com/mschauer/Bridge.jl/releases/tag/v0.4.0)
Travis: [Travis Build Status](https://travis-ci.org/mschauer/Bridge.jl/builds/247967697)
cc: @mschauer

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

@mschauer This PR will remain open for 24 hours for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

**Comment by mschauer** [comment]
This is Bridge.jl, a statistical toolbox for diffusion processes written in julia.

The package allows for simulation of univariate and multivariate stochastic processes in continuous time, and also from conditional diffusion processes (similar to Brownian bridges, namesakes of this package)

Several examples illustrate how the functions in this package can be used in Bayesian inference for drift and diffusion coefficients using the Metropolis-Hastings algorithm, see F. v. d. Meulen, M. Schauer: Bayesian estimation of discretely observed multi-dimensional diffusion processes using guided proposals. Electronic Journal of Statistics 11 (1), 2017, 10.1214/17-EJS1290.

**Comment by tkelman** [comment]
Not a package about the card game bridge?

**Comment by mschauer** [comment]
The name was shortly discussed in https://github.com/mschauer/Bridge.jl/pull/2#issuecomment-311645963 with the conclusion to keep it.

File: Bridge/versions/0.4.0/requires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+julia 0.6
+Compat
```
Comment (tkelman) [inline Bridge/versions/0.4.0/requires]: It's best to declare a minimum version dependency that implements the features you need.

Comment (mschauer) [inline Bridge/versions/0.4.0/requires]: I don't have a good overview over that, honestly.

Comment (tkelman) [inline Bridge/versions/0.4.0/requires]: are you using Compat at all if it's 0.6-only?

Comment (mschauer) [inline Bridge/versions/0.4.0/requires]: No! :-)

Comment (tkelman) [inline Bridge/versions/0.4.0/requires]: may as well remove it for now then?

Comment (mschauer) [inline Bridge/versions/0.4.0/requires]: yes, I am about to commit this

**Comment by mschauer** [comment]
Close.

**Pull request closed without merging**